### PR TITLE
[7884] remove unnecessary admin fields from mozfest homepage

### LINKED ISSUE #7883: mozfest nav button should always be editable in the CMS
The recent video work for the mozfest homepage introduced some logic that hides certain fields that are not needed in the CMS edit view for the page, but it takes out too much. Let's:

- [x] restore "all panels" first (https://github.com/mozilla/foundation.mozilla.org/pull/7884/files)
- [x] redo the logic that decides which fields to keep for which view, with @marcwalsh to confirm

### PR DESCRIPTION
**Pull request by richbrennan**
Hi @Pomax , happy new year! :-)

Closes #7883

Specifies the admin fields on the mozfest homepage content panels, as per Marc's list: https://github.com/mozilla/foundation.mozilla.org/issues/7883#issuecomment-996039763

I didn't attempt to re-implement the dynamic `content_panels` generation from https://github.com/mozilla/foundation.mozilla.org/pull/7884/files because it was dependant on a hard-coded value, so doesn't need conditional logic, but correct me if I'm wrong :-)

**Link to sample test page**: https://foundation-s-7883-mozfe-ivs7ow.herokuapp.com/cms/pages/161/edit/

### CODE REVIEW

**Comment by richbrennan** [comment]
> Hey @richbrennan, just a heads up that the changes need to be replicated to the `translatable_fields` variable below.
> 
> Thanks!

Thanks for spotting Theo! I've updated the  `translatable_fields`

File: network-api/networkapi/mozfest/models.py [hunk 1]
```diff
@@ -208,21 +208,18 @@ class MozfestHomepage(MozfestPrimaryPage):
...
 
-    # Put everything above the body
-    parent_panels = MozfestPrimaryPage.content_panels
-    panel_count = len(parent_panels)
-    n = panel_count - 1
-
-    content_panels = parent_panels[:n] + [
-        FieldPanel('cta_button_label'),
-        FieldPanel('cta_button_destination'),
+    content_panels = Page.content_panels + [
```
Comment (Pomax) [inline network-api/networkapi/mozfest/models.py:211]:
```suggestion
    # See https://github.com/mozilla/foundation.mozilla.org/issues/7883#issuecomment-996039763
    content_panels = Page.content_panels + [
```

**Comment by Pomax** [comment]
And a happy new year to you, too!

**Review by Pomax — APPROVED** [review]
added a comment that explains why we have the non-inheriting set of content panels, but with that: R+

**Review by TheoChevalier — APPROVED** [review]
Thanks! Looks good

**Pull request merged**